pkg/daemon/update.go

```diff
@@ -690,6 +690,17 @@ func (dn *Daemon) updateOS(config *mcfgv1.MachineConfig) error {
 	return nil
 }
 
+// RHEL 7.6 logger (util-linux) doesn't have the --journald flag
+func loggerWithoutJournalOption() bool {
+	loggerOutput, err := exec.Command("logger", "--journald").CombinedOutput()
```

Comment:
I think we should use `logger --help`...as running this command just hangs for me; maybe it works because we're not providing stdin?

Or better, do:

```
if dn.OperatingSystem == machineConfigDaemonOSRHCOS {
    return true
```
?

Reply:
uhm, does it hang on RHCOS? works fine on rhel7.6 :thinking:

Reply:
oh yeah I get it now thanks!

Reply:
fixed